### ISSUE TITLE
about Masked Feature Prediction for Self-Supervised Visual Pre-Training

### ISSUE THREAD

**Issue by cquzys** [facebookresearch/pytorchvideo#192]
Hi author:
   Can you share your codes of paper "Masked Feature Prediction for Self-Supervised Visual Pre-Training"?
    Thanks

**Comment by Wallace-222** [comment]
Any updates?

**Comment by lyttonhao** [comment]
Hi, the official MaskFeat have been released in [PySlowFast](https://github.com/facebookresearch/SlowFast/tree/main/projects/maskfeat). We might also support MaskFeat in PytorchVideo in the future.
